src/renderer/index.html

```diff
@@ -32,5 +32,7 @@
     <nwb-dashboard></nwb-dashboard>
 
     <script type="module" src="./src/index"></script>
+    <script async src="https://w.appzi.io/w.js?token=wR0Rh"></script>
```

Comment:
```suggestion
    <script async src="https://w.appzi.io/w.js?token=hUmCm"></script>
```

Changing token to the one for the NWB GUIDE account.

Reply:
Awesome. Thanks for taking the lead on this!